Source/Initialization/WarpXInitData.cpp

```diff
@@ -1144,7 +1145,7 @@ WarpX::InitializeExternalFieldsOnGridUsingParser (
                 const amrex::Real x = 0._rt;
                 const amrex::Real y = 0._rt;
                 const amrex::Real fac_z = (1._rt - x_nodal_flag[0]) * dx_lev[0] * 0.5_rt;
-                const amrex::Real z = i*dx_lev[0] + real_box.lo(0) + fac_z;
+                const amrex::Real z = j*dx_lev[0] + real_box.lo(0) + fac_z;
```

Comment:
Is this change (`i` -> `j`) on purpose? I think it should be `i` since in 1d only the first index is used.

Reply:
Good catch! That's a typo, I'll fix.